packages/cache/src/internal/cacheUtils.ts

```diff
@@ -71,11 +71,15 @@ export async function unlinkFile(filePath: fs.PathLike): Promise<void> {
   return util.promisify(fs.unlink)(filePath)
 }
 
-async function getVersion(app: string): Promise<string> {
-  core.debug(`Checking ${app} --version`)
+async function getVersion(
+  app: string,
+  additionalArgs: string[] = []
+): Promise<string> {
   let versionOutput = ''
+  additionalArgs.push('--version')
+  core.debug(`Checking ${app} ${additionalArgs.join(' ')}`)
   try {
-    await exec.exec(`${app} --version`, [], {
+    await exec.exec(`${app}`, additionalArgs, {
```

Comment:
Thanks again @pdotl!

Just saw this while looking at the PR. `'${app}'` seems a bit redundant. The first argument should be a string and `app` is already one. Maybe you can fix that when you address the `zstd --long` issue. It would be a bit much to open a separate PR for it 😅

Reply:
Sure @cdce8p will take this up in the next PR.